tests/unit/withOnyxTest.js

```diff
@@ -25,7 +25,7 @@ describe('withOnyx', () => {
     it('should render with the test data when using withOnyx', () => {
         let result;
 
-        Onyx.set(ONYX_KEYS.TEST_KEY, 'test1')
+        return Onyx.set(ONYX_KEYS.TEST_KEY, 'test1')
```

Comment:
There are some tests here that need to be double checked
This one didn't return the promise so it always passed successfully